include/sdfg/codegen/instrumentation/instrumentation_plan.h

```diff
@@ -3,36 +3,32 @@
 #include <unordered_map>
 #include "sdfg/analysis/analysis.h"
 #include "sdfg/analysis/flop_analysis.h"
+#include "sdfg/codegen/instrumentation/instrumentation_info.h"
 #include "sdfg/codegen/language_extension.h"
 #include "sdfg/codegen/utils.h"
+#include "sdfg/data_flow/library_node.h"
+#include "sdfg/element.h"
 #include "sdfg/structured_control_flow/control_flow_node.h"
 #include "sdfg/structured_sdfg.h"
 #include "sdfg/symbolic/symbolic.h"
+#include "sdfg/visitor/immutable_structured_sdfg_visitor.h"
 
 namespace sdfg {
 namespace codegen {
 
-enum InstrumentationEventType {
-    CPU = 0,
-    CUDA = 1,
-};
+enum InstrumentationEventType { CPU = 0, CUDA = 1, TENSTORRENT = 2, H2D = 3, D2H = 4 };
```

Comment:
They map to the RTL Event sets: CPU, CUDA, NONE. Why not reflect this here? Just confusing if they have a different meaning. Also the .cpp does not use them